docs/manual/java/guide/cluster/PersistentEntityAdvanced.md

```diff
@@ -0,0 +1,58 @@
+# Advanced Persistent Entity Usage
+
+Below are some advanced usages of persistent entities.
+
+## Child actors
+
+Lagom's `PersistentEntityRegistry` creates and distributes persistent entities by running them in actors, distributed across a Lagom cluster using [Akka cluster sharding](https://doc.akka.io/docs/akka/current/cluster-sharding.html?language=java). In some use cases, it may make sense to run persistent entities as a child actor of a user managed actor, which itself may be distributed using cluster sharding. Some examples of such use cases include:
+
+* Persistent process managers. When you have a long running operation that involves multiple interactions with other systems or entities, it is often necessary to store the state of the process, so that it can be continued by an asynchronous signal (such as a message arriving through a message queue). Lagom's persistent entity API can be a good fit for this persistence, since it provides an audit log of the process, and it allows read side processors to do reconciliation on the process.
```

Comment:
I think it would be good to extend this explanation to detail how the introduced feature fits this need. Having an example that required a `ChildPersistentEntity` interacting with other `PersistentEntity` or remote endpoints.

Reply:
I think that would be better in a blog post or other technical article than in the reference documentation. Any practical use case is going to be very specific to that actual use case, whereas this documentation I'd think would be more about how to use the feature in general. Also, describing a process manager in full will probably require having a sample app to back it up for reference. I don't think what we can feasibly put in the documentation will be enough to adequately describe the full use case.